### DISCUSSION TITLE
Limiting Case 2 Hint

### DISCUSSION THREAD

**Discussion by Star-Ruby** [Part-II-Computational-Physics/.github#70] [Q&A] [answered]
Hint: The system will equilibrate much faster if you start with an initial state that is close to the expected final state.

What is meant by this? How do we start with a state like that?

**Comment by DLHvdH** [comment, marked as answer]
The analytical solution tells you if the expected final magnetisation is closer to 1, 0, or -1. You can start your Monte Carlo simulation with an average magnetisation closest to that value.
